content/ecosystem/bridges/discord/bridges.toml

```
@@ -93,14 +93,14 @@ typing_notifications = true
 [[bridges]]
 name = "Out Of Your Element"
 maintainer = "Cadence"
-summary = "An efficient and modern bridge. It supports the latest Discord features like replies and threads, and the latest Matrix features like custom emotes and intentional mentions."
-maturity = "Beta"
+summary = "An efficient and modern bridge. It supports the latest Discord features like replies and threads, and the latest Matrix features like custom emotes and authenticated media. Easy to configure with web UI."
+maturity = "Stable"
 language = "JavaScript"
 license = "AGPL-3.0-or-later"
 docs = "https://gitdab.com/cadence/out-of-your-element/src/branch/main/docs"
 repo = "https://gitdab.com/cadence/out-of-your-element"
 room = "#out-of-your-element:cadence.moe"
-featured = false
+featured = true
```

Sorry, we currently don't have a process for deciding which projects to feature, so for now we have to go without.
The website working group plans to work on a policy for this and similar questions.
```suggestion
featured = false
```

---

okeydokey, thank you :)